README.md

```diff
@@ -141,6 +170,40 @@ export CUSTOM_SEARCH_ENGINE_ID="YOUR_CUSTOM_SEARCH_ENGINE_ID"
 
 ```
 
+## Redis Setup
```

Comment:
I mean things like this. Redis setup is clearly unrelated

Reply:
Please comment on https://github.com/Torantulino/Auto-GPT/pull/142

Reply:
Idk how this snuck in to be honest, thanks for the catch.